scripts/qemu/run-build.sh

```diff
@@ -34,17 +34,3 @@ cmake .. \
     -DUMF_BUILD_EXAMPLES=ON
 
```

Comment:
pls add `-DUMF_TESTS_FAIL_ON_SKIP=ON` - right now we missed the skipped ipc test

Reply:
done